code/core/src/types/modules/addons.ts

```diff
@@ -398,7 +398,7 @@ export interface Addon_PageType {
    *
    * If you want to render your component only when the URL matches, use the `Route` component.
    * @example
-   * import { Route } from '@storybook/router';
+   * import { Route } from '@storybook/core/router';
```

Comment:
Shouldn't this be `storybook/router` instead?

Reply:
Do you think it's wise to do that? because a references to `storybook` from `@storybook/core` would be a cyclical reference.

In this case it's just a comment, so no harm.
But it's hard to know what exactly to write here, because it depends on where the comment is read/used.

I'm included to stay consistent and never reference the `storybook` package from the `@storybook/core` package.

Reply:
My mistake. Got confused. You're right!